src/symbolize/dbghelp.rs

```diff
@@ -10,8 +10,12 @@
 
 #![allow(bad_style)]
 
+#[cfg(feature = "std")]
+use std::char;
+
 use core::mem;
 use core::slice;
+#[cfg(not(feature = "std"))]
```

Comment:
Hm what's going on here? Was this something we moved from std to core in the last few versions?

Reply:
Yeap, based on issue https://github.com/rust-lang/rust/issues/49319 and the corresponding PR: https://github.com/rust-lang/rust/pull/49698 which happened on 2018-04-12, which is *just* after Rust 1.26 was [released](https://github.com/rust-lang/rust/blob/master/RELEASES.md#version-1260-2018-05-10).